openslides_backend/action/actions/motion/set_state.py

```diff
@@ -22,6 +22,47 @@ class MotionSetStateAction(UpdateAction, SetNumberMixin, PermissionHelperMixin):
     model = Motion()
     schema = DefaultSchema(Motion()).get_update_schema(["state_id"])
 
+    """
```

Comment:
This comment is for you, because the taken decisions could be reflected.
I would prefer to have for all actions this kind of documentation to accelerate the action.
I wrote a threaded test for motion.set_state and with 500 threads in parallel it lasts 28,33 sec with reduced amount of locked_fields, the original version 31,53 sec..
And the duration time is nearly linear, 5.3sec for 100, 31.53sec for 500 motion.set_state actions.

Reply:
The problem with such docs is that they must be kept up-to-date to be meaningful. Since they are highly specific (comparable to unit tests), they can potentially be needed to update with each code logic change. I don't think the improved value is worth this work. Also, my remark still stands - it doesn't make much sense to keep this in the code. Maybe just post it in the issue/PR next time?

But the timing sounds ok. We have to be careful, though - missing a required lock somewhere can result in a corrupt database. Since the gain is not that high, we should rather be safe than sorry on important places.

Reply:
comment removed